README.md

```diff
@@ -157,28 +207,37 @@ curl -v -XGET -H "Authorization: Bearer $(ocm token)" http://localhost:8000/api/
 # Delete a kafka request
 curl -v -X DELETE -H "Authorization: Bearer $(ocm token)" http://localhost:8000/api/kafkas_mgmt/v1/kafkas/<kafka_request_id>
 ```
+
 #### Using RHOAS CLI to manage Kafka instances
 
-The locally installed kas-fleet-manager doesn't deploy TLS enabled kafka admin server but the default URL scheme used by the app-service-cli is HTTPS. So, the url scheme should be changed to http in the CLI [code](https://github.com/redhat-developer/app-services-cli/blob/main/pkg/core/connection/api/defaultapi/default_client.go#L155) and it should be built locally
-```
-make binary
-```
-Then, it can be pointed towards the locally running fleet manager
+The locally installed kas-fleet-manager doesn't deploy TLS enabled kafka admin
+server but the default URL scheme used by the app-service-cli is HTTPS. So,
+the url scheme should be changed to http in the
+CLI [code](https://github.com/redhat-developer/app-services-cli/blob/main/pkg/core/connection/api/defaultapi/default_client.go#L155)
+and it should be built locally.
+
+First [run the Fleet Manager locally](#running-the-service-locally)
+
+Then, RHOAS CLI can run pointing to the locally running Fleet Manager:
 ```
 ./rhoas login --mas-auth-url=stage --api-gateway=http://localhost:8000
-``` 
-Now, various kafka specific operations can be performed as described [here](http://appservices.tech/commands/rhoas_kafka)
+```
+
+Now, various kafka specific operations can be performed
+as described [here](http://appservices.tech/commands/rhoas_kafka)
 
 #### Using the Kafka admin-server API
- - The admin-server API is used for managing topics, acls, and consumer groups. The API specification can be found [here](https://github.com/bf2fc6cc711aee1a0c2a/kafka-admin-api/blob/main/kafka-admin/src/main/resources/openapi-specs/kafka-admin-rest.yaml)
-- To get the API endpoint, use the following command
-```
-oc get routes -n kafka-c7ndprea8ueq4mmrm3c0 | grep admin-server
 
-# Here c7ndprea8ueq4mmrm3c0 is the kafka instance ID.
-```
+* The admin-server API is used for managing topics, acls, and consumer groups.
+  The API specification can be found [here](https://github.com/bf2fc6cc711aee1a0c2a/kafka-admin-api/blob/main/kafka-admin/src/main/resources/openapi-specs/kafka-admin-rest.yaml)
+* To get the API endpoint, use the following command
+    ```
+    # Here c7ndprea8ueq4mmrm3c0 is the kafka instance ID
+    oc get routes -n kafka-c7ndprea8ueq4mmrm3c0 | grep admin-server
+    ```
```

Comment:
This is not needed anymore. 
The admin-serve endpoint is returned as part of the Kafka GET/LIST endpoints as soon as the Kafka is ready - https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/acddfc4de0fa99fc22a268f1a3e0fdf135cb7745/openapi/kas-fleet-manager.yaml#L983

Reply:
This PR is about formatting changes but I will change it anyway :+1:

Reply:
Thanks

Reply:
Changed